openwisp_controller/config/api/filters.py

```diff
@@ -142,3 +143,18 @@ def __init__(self, *args, **kwargs):
 
     class Meta(BaseConfigAPIFilter.Meta):
         model = DeviceGroup
+
+
+class ReversionFilter(BaseConfigAPIFilter):
+    model = filters.CharFilter(field_name="content_type__model", lookup_expr="iexact")
+
+    def _set_valid_filterform_labels(self):
+        self.filters["model"].label = _("Model")
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._set_valid_filterform_labels()
+
+    class Meta:
+        model = Version
+        fields = ["model"]
```

Comment:
Please use single quotes here and everywhere (we'll switch to double quotes later but we haven't done so yet so please maintain consistency).